cypress/integration/login_spec.js

```diff
@@ -0,0 +1,59 @@
+import helpers from '../helpers/helpers'
+
+describe('Login', function() {
+  beforeEach(() => {
+    cy.visit(`${helpers.getStoryUrl(['Apps', 'Login'], 'Login')}`)
+  })
+
+  describe('Login', () => {
+    it('should load and display login view', function() {
+      cy.contains('Dieser Bereich ist privat.', {timeout: 15000})
+      cy.get('button').should('have.length', 2)
+    })
+
+    it('should not be possible to click login button', function() {
+      cy.get('[data-cy=login-form_user-input]')
+        .type('{selectall}{del}cypress_test')
+      cy.get('[data-cy=login-form_login-button]')
+        .should('be', 'disabled')
+      cy.contains('Dieser Bereich ist privat.')
+      cy.get('[data-cy=login-form_user-input]')
+        .type('{selectall}{del}')
+      cy.get('[data-cy=login-form_password-input]')
+        .type('{selectall}{del}12345')
+      cy.get('[data-cy=login-form_login-button]')
+        .should('be', 'disabled')
+      cy.contains('Dieser Bereich ist privat.')
+    })
+
+    it('should login', function() {
+      cy.get('[data-cy=login-form_user-input]')
+        .type('{selectall}{del}cypress_test')
+      cy.get('[data-cy=login-form_password-input]')
+        .type('{selectall}{del}Test_pw1')
+      cy.get('[data-cy=login-form_login-button]')
+        .click()
+        .children().first().should('be', 'svg', {timeout: 5000})
+    })
+
+    it('should request password from request page', function() {
+      cy.contains('Passwort vergessen?')
+      cy.get('[data-cy=login-form_request-button]')
+        .click()
+      cy.get('[data-cy=password-request_input]')
+        .type('{selectall}{del}test@testmail.com')
+      cy.get('[data-cy=password-request_submit-button]')
+        .click()
+      cy.contains('Passwort wurde per E-Mail versendet')
+    })
+
+    it('should change to login page on abort of password request ', function() {
+      cy.get('[data-cy=login-form_request-button]')
+        .click()
+      cy.contains('Passwort vergessen?')
+      cy.get('[data-cy=password-request_abort-button]')
+        .click()
+      cy.contains('Dieser Bereich ist privat.')
```

Comment:
Isn't there a more reliable assertion? Can't you, for example, expect the username and password field to be visible? (selected by `data-cy` attribute)